packages/core/src/test/awsService/cloudWatchLogs/commands/tailLogGroup.test.ts

```diff
@@ -39,23 +36,12 @@ describe('TailLogGroup', function () {
     let cloudwatchSettingsSpy: sinon.SinonSpy
     let wizardSpy: sinon.SinonSpy
 
-    let clock: FakeTimers.InstalledClock
-
-    before(function () {
-        clock = installFakeClock()
```

Comment:
Using fake clocks is screwing with the `waitUntil()` calls. Inspecting the test that uses the clock: the behavior under test is a functionality of `LiveTailSession`, and not `TailLogGroup`. If we want to assert that the timer is disposed of, it should be done in those unit tests, and not here. 

Removing the clock.